libs/NfcCoreLib/lib/LibNfc/phLibNfc_Se.c

```diff
@@ -271,6 +281,56 @@ NFCSTATUS phLibNfc_SE_SetMode ( phLibNfc_Handle              hSE_Handle,
     return wStatus;
 }
 
+NFCSTATUS phLibNfc_SE_PowerAndLinkControl(phLibNfc_Handle              hSE_Handle,
+                                          phLibNfc_PowerLinkModes_t    powerLinkModes,
+                                          pphLibNfc_RspCb_t            pSE_PowerAndLinkControl_Rsp_cb,
+                                          void *                       pContext
+                                         )
+{
+    NFCSTATUS      wStatus = NFCSTATUS_SUCCESS;
+    pphLibNfc_Context_t pCtx = phLibNfc_GetContext();
+    phLibNfc_Event_t TrigEvent = phLibNfc_EventDummy;
+    phLibNfc_DummyInfo_t Info;
+    PH_LOG_LIBNFC_FUNC_ENTRY();
+
+    wStatus = phLibNfc_IsInitialised(pCtx);
+
+    if (NFCSTATUS_SUCCESS != wStatus)
+    {
+        PH_LOG_LIBNFC_CRIT_STR("LibNfc Stack not Initialised");
+        wStatus = NFCSTATUS_NOT_INITIALISED;
+    }
+    else if ((NULL == (void *)hSE_Handle) ||
+             (NULL == pSE_PowerAndLinkControl_Rsp_cb))
+    {
+        wStatus = NFCSTATUS_INVALID_PARAMETER;
+    }
+    else if (phNciNfc_IsVersion2x(pCtx->sHwReference.pNciHandle))
+    {
+        Info.Evt = phLibNfc_DummyEventPowerAndLinkCtrl;
+
+        Info.Params = (void *)&powerLinkModes;
+        wStatus = phLibNfc_StateHandler(pCtx,
+                                        TrigEvent,
+                                        (void *)hSE_Handle, /*Secure Element Handle*/
+                                        (void *)&Info, /*Information for setting Power and Link for SE*/
+                                        NULL);
+        if (NFCSTATUS_PENDING == wStatus)
+        {
+            /*Store CB info in SE Context*/
+            pCtx->CBInfo.pPowerCtrlLinkCb = pSE_PowerAndLinkControl_Rsp_cb;
+            pCtx->CBInfo.pPowerCtrlLinkCntx = pContext;
+        }
+    }
+    else
+    {
+        PH_LOG_LIBNFC_CRIT_STR("NFCC is not NCI2.0 compliant!");
```

Comment:
nit: "NFCC is not operating in NCI2.0 mode" is probably more accurate.

Reply:
Change is now updated reflecting this